src/main/scala/io/iohk/ethereum/db/dataSource/IodbDataSource.scala

```diff
@@ -1,23 +1,46 @@
 package io.iohk.ethereum.db.dataSource
 
+import java.io.File
 import java.nio.ByteBuffer
 import java.util.concurrent.atomic.AtomicLong
 
 import io.iohk.iodb.{ByteArrayWrapper, LSMStore}
 
-class IodbDataSource(lSMStore: LSMStore) extends DataSource {
+class IodbDataSource private (lSMStore: LSMStore,
+                              keySize: Int,
+                              path: String) extends DataSource {
```

Comment:
actually @rtkaczyk i think it's a good comment. The problem I see it's probably around other places (did a quick search and found some)....mayte it's better to create a separate task for this, what do you think?